gstroup/apimocker
# Switch from jshint to eslint and delint using airbnb styles

This swaps out `jshint` with `eslint` as a step toward updating the repository to use ES6 and targeting later versions of Node.js.

[eslint](https://eslint.org/) provides more capabilities than jshint, and supports style presets such as the popular airbnb styles.

All files have also been delinted to comply with the new styling rules. **No functional changes have been made--only syntactic and whitespace updates.** This should improve the consistency of the JS.

I recognize this is a significant stylistic change, and style is subjective, so I will not be offended if the current style is preferred over `airbnb`'s.

### To verify
Since delinting has modified a lot of files, it's best to pull the branch and view the output locally rather than within github.

1. `npm run lint` (or `grunt eslint`)
_Expected results_
Linting should pass and exit without an error code.

2. `npm test`
_Expected results_
All tests should pass.
